plugins/lightspeed-backend/src/service/router.test.ts

```diff
@@ -1,6 +1,4 @@
-import { getVoidLogger } from '@backstage/backend-common';
-// import { mockCredentials, mockServices } from '@backstage/backend-test-utils';
-import { MockConfigApi } from '@backstage/test-utils';
+import { mockServices } from '@backstage/backend-test-utils';
```

Comment:
This test file should be refactored to follow the structure of the ocm `router.test.ts` file.